src/analysis/retail/mage/frost/core/BrainFreeze.tsx

```diff
@@ -70,7 +57,7 @@ class BrainFreeze extends Analyzer {
   }
 
   get overlappedFlurries() {
-    return this.flurry.filter((f) => f.overlapped).length;
+    return this.flurry.flurryEvents.filter((f) => f.overlapped).length;
   }
```

Comment:
should this still live on `BrainFreeze` or ought it move to Flurry?

Reply:
yes, I think you're right. Maybe it belongs to the flurry module. I'll look into it.

Reply:
I'm currently working on the brain freeze module, if it's ok with you I can do this change on the next PR, to avoid merge conflicts between this branch and the branch I working on now. Makes sense ?